src/js/components/account/user-registration.js

```diff
@@ -163,7 +190,9 @@ export class UserRegistration {
           $fail[0].scrollIntoView({ behavior: "smooth" });
         } catch (e) {
           console.error(e);
-          alert(`An error occurred. Please try again or contact support at ${SUPPORT_EMAIL}`);
+          alert(
+            `An error occurred. Please try again or contact support at ${SUPPORT_EMAIL}`
+          );
```

Comment:
Could we use the same in-page template we use on other pages to show an error instead of the `alert`?

Reply:
It looks like this is a try catch statement that only shows the alert if rendering the in-page template fails for whatever reason.

I didn't add this extra try-catch so I'm not going to remove it as it might be there for reason, considering this is a fail-over for rendering the in-page template I think it's fine to leave as an alert